task/tkn/0.2/tkn.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     tekton.dev/pipelines.minVersion: "0.17.0"
     tekton.dev/tags: cli
     tekton.dev/displayName: "Tekton CLI"
+    tekton.dev/platforms: "linux/amd64,linux/s390x,linux/ppc64le"
```

Comment:
should we add the same label for 0.1 as well?

Reply:
done for git-cli and tkn tasks